web/components/PetPostModal/ImageSlider.tsx

```diff
@@ -50,28 +50,49 @@ function ImageSlider(props: { attachments: Array<string> }) {
         );
       }}
     >
-      {/* TODO: Account for Videos */}
       {props.attachments.length > 0
         ? props.attachments?.map((slide) => {
             return (
               <Flex
                 justifyContent={"center"}
                 alignItems={"center"}
-                bgColor={"tag-primary-bg"}
+                bgColor={
+                  slide.slice(slide.length - 3) === "mp4"
+                    ? "black"
+                    : "tag-primary-bg"
```

Comment:
It's not guaranteed that the file extension is 3 characters long. Split by `.` instead and get the last element for the file extension. The video formats we support are `.mp4` and `.mov`, but you need to make sure these are case insensitive. In other places in the app, we've instantiated a set `new Set(["mp4", "mov"])` and we use `set.has(___.toLowerCase())` to check.